cmd/system-probe/subcommands/usm/debug_scripts/usm_leak_detector/backends/bpftool.py

```diff
@@ -0,0 +1,153 @@
+"""
+bpftool backend for eBPF map operations.
+"""
+
+import json
+import subprocess
+from typing import Dict, Generator, List, Optional
+
+from .ebpf_backend import EbpfBackend
+from ..logging_config import logger
+from .bpftool_downloader import download_bpftool
+from .streaming import iter_json_objects
+from ..subprocess_utils import safe_subprocess_run
+
+
+class BpftoolBackend(EbpfBackend):
+    """eBPF backend using bpftool."""
+
+    def __init__(self, bpftool_path: str = "bpftool"):
+        """Initialize with optional custom bpftool path."""
+        self._bpftool_path = bpftool_path
+
+    @staticmethod
+    def name() -> str:
+        return "bpftool"
+
+    @staticmethod
+    def is_available(try_download: bool = True) -> bool:
+        """Check if bpftool is available, optionally downloading it."""
+        # First try system bpftool
+        try:
+            result = safe_subprocess_run(
+                ["bpftool", "version"],
+                capture_output=True,
+                timeout=5
```

Comment:
lets create also for the entire PR timeout hard coded value which will be clear to us

Reply:
DEFAULT_SUBPROCESS_TIMEOUT?